pom.xml

```diff
@@ -140,4 +140,24 @@
 
   </profiles>
 
+  <!-- TODO move this into kie-bom -->
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>kie-wb-distribution-wars</artifactId>
+        <classifier>wildfly10</classifier>
+        <version>${version.org.kie}</version>
+        <type>war</type>
+      </dependency>
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>kie-drools-wb-distribution-wars</artifactId>
+        <version>${version.org.kie}</version>
+        <classifier>wildfly10</classifier>
+        <type>war</type>
+      </dependency>
```

Comment:
This is already fixed in the kie-bom, see https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/pull/218/commits/eb5afc91fc500af5b0fad4e67bbcd2a1afa79325. I will remove it from here.